sdk/servicebus/service-bus/test/serviceBusClient.spec.ts

```diff
@@ -63,7 +62,7 @@ describe("Random scheme in the endpoint from connection string", function(): voi
     sbClientWithRelaxedEndPoint = new ServiceBusClient(
       getEnvVars().SERVICEBUS_CONNECTION_STRING.replace("sb://", "CheeseBurger://")
     );
-    sender = sbClientWithRelaxedEndPoint.createSender(entities.queue!);
+    sender = await sbClientWithRelaxedEndPoint.createSender(entities.queue!);
```

Comment:
```suggestion
    sender = sbClientWithRelaxedEndPoint.createSender(entities.queue!);
```
(it's not async anymore)

Reply:
Thanks!